.github/ISSUE_TEMPLATE/usage-question----insert-question-here--.md

```diff
@@ -0,0 +1,14 @@
+---
+name: 'Usage Question: {{Insert Question here}}'
+about: General Questions about ArviZ Usage
+
+---
+
+## Short Description
+Let us know what you're trying to do and we can point you in the right direction
+
+## Code Example or link
+Please insert an example demonstrating what you're trying to do. Ideally it would be a link to a notebook or code repository that can be run on another user's computer
```

Comment:
Please provide a minimal, self-contained, and reproducible example demonstrating what you're trying to do. Ideally...